app/assets/src/components/views/SampleView/SampleView.jsx

```diff
@@ -19,6 +19,7 @@ import ViewHeader from "~/components/layout/ViewHeader";
 import NarrowContainer from "~/components/layout/NarrowContainer";
 import Tabs from "~/components/ui/controls/Tabs";
 import DetailsSidebar from "~/components/common/DetailsSidebar";
+import CoverageVizSidebar from "~/components/common/CoverageVizSidebar";
```

Comment:
I tend to associate Sidebar to Left-Right... I wonder if makes more sense to call it something that indicates it is used in the bottom side.